spec/lib/config_spec.rb

```diff
@@ -47,6 +60,19 @@ module Fusuma
       }
     end
 
+    let(:keymap_with_trigger_interval) do
+      {
+        'swipe' => {
+          4 => {
+            'right'  => {
```

Comment:
__[RuboCop]__

Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space.


<!-- Comment by [Sider](https://sider.review) -->